examples/reference-erc4626/src/index.ts

```diff
@@ -0,0 +1,83 @@
+import { ponder } from "@/generated";
+
+ponder.on("ERC4626:Deposit", async ({ event, context }) => {
+  // keep assets and shares count for account
```

Comment:
Should be removed, or a complete sentence like `Keep assets and shares count for account.`